From 6648f6b684a09efd5598e5a709223992bdebfa12 Mon Sep 17 00:00:00 2001
From: Takashi Kajinami <tkajinam@redhat.com>
Date: Mon, 18 Apr 2022 12:50:38 +0900
Subject: [PATCH] Bump minimum version of gnocchiclient for aggregats API
 support

The commit 74eadfbd58359b7ebe9e1e40ae6b6ff245146bb8 introduced usage
of aggregates API, which is available since gnocchiclient 6.0.0 .

This change bumps the minimum version to ensure the API is available.

Change-Id: I64a984fe64c8fe69856980df9e28a67f5eb29b58
---
 requirements.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/requirements.txt b/requirements.txt
index f5a55cba..63f0b1fc 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -7,7 +7,7 @@ croniter>=0.3.4 # MIT License
 futurist>=0.11.0  # Apache-2.0
 jsonschema>=3.2.0 # MIT
 keystonemiddleware>=5.1.0 # Apache-2.0
-gnocchiclient>=3.1.0  # Apache-2.0
+gnocchiclient>=6.0.0  # Apache-2.0
 lxml>=2.3
 oslo.db>=4.8.0,!=4.13.1,!=4.13.2,!=4.15.0 # Apache-2.0
 oslo.config>=6.8.0 # Apache-2.0
-- 
2.37.2

